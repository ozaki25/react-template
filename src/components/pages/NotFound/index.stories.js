import React from 'react';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';
import NotFound from '.';

const stories = storiesOf('pages', module);

stories.add('NotFound', withInfo('404ページ')(() => <NotFound />));
