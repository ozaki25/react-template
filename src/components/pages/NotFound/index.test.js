import React from 'react';
import { snapshot } from '../../../../test/helpers';
import NotFound from './';

snapshot('NotFound', <NotFound />);
