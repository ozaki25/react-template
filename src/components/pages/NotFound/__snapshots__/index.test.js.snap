// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: NotFound 1`] = `
<NotFound>
  <h2>
    NotFound
  </h2>
</NotFound>
`;
